shopify_python/google_styleguide.py

```diff
@@ -251,3 +256,48 @@ def __use_cond_expr(self, node):  # type: (astroid.If) -> None
                 else_body_name = node.orelse[0].targets[0].name
                 if if_body_name == else_body_name:
                     self.add_message('cond-expr', node=node)
+
+    def __lambda_func(self, node):  # type: (astroid.Lambda) -> None
+        """Prefer Operator Function to Lambda Functions"""
+
+        if isinstance(node.body, astroid.UnaryOp):
```

Comment:
@zeedann Here is the guideline you are implementing:
> use the functions from the operator module instead of lambda functions

Here's the case we are discussing:
```python
f = lambda x: x * 3
```
What is the example code that you are proposing?

```python
f = lambda x: operator.mul(x, 3)
```

Why is this better? It is not replacing a lambda with anything. It's replacing a bare operator with a function call.

Here is an alternative that replaces the lambda:

```python
f = functools.partial(operator.mul, 3)
```

But that's a bad idea :)